src/Razor/Microsoft.NET.Sdk.Razor/src/build/netstandard2.0/Microsoft.NET.Sdk.Razor.CssIsolation.targets

```diff
@@ -0,0 +1,90 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.Sdk.Razor.StaticWebAssets.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+
+<Project ToolsVersion="14.0">
+
+<UsingTask TaskName="Microsoft.AspNetCore.Razor.Tasks.RewriteCss" AssemblyFile="$(RazorSdkBuildTasksAssembly)" />
+<UsingTask TaskName="Microsoft.AspNetCore.Razor.Tasks.ConcatenateCssFiles" AssemblyFile="$(RazorSdkBuildTasksAssembly)" />
+
+<PropertyGroup>
+  <RazorGenerateComponentDeclarationDependsOn>$(RazorGenerateComponentDeclarationDependsOn);ResolveIsolatedCss</RazorGenerateComponentDeclarationDependsOn>
+  <ResolveIsolatedCssDependsOn>ResolveRazorComponentInputs;ResolveCssFiles;GenerateScopedCssFiles</ResolveIsolatedCssDependsOn>
```

Comment:
I was listening to Rainer's MSBuild talk recently and they pretty much recommend using `Before` \ `After` targets when possible over properties. Just something to think about.

Reply:
I want to keep consistency with our current SDK implementation